src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -824,6 +832,54 @@ public function iShouldNotSeeInformationAboutShipments()
         Assert::same($this->showPage->getShipmentsCount(), 0);
     }
 
+    /**
+     * @Then the product's :productName unit price should be :price
+     */
+    public function productUnitPriceShouldBe(string $productName, string $price): void
+    {
+        Assert::same($this->showPage->getItemUnitPrice($productName), $price);
+    }
+
+    /**
+     * @Then the product's :productName item discount should be :price
+     */
+    public function productItemDiscountShouldBe(string $productName, string $price): void
```

Comment:
```suggestion
    public function productsItemDiscountShouldBe(string $productName, string $price): void
```

Reply:
i intentionally deleted 's' from method name because I don't want to suggest a plural form in its name.